assets/eip-7265/src/utils/LimiterLib.sol

```diff
@@ -0,0 +1,136 @@
+// SPDX-License-Identifier: MIT
```

Comment:
Assuming you have the authority to change the license, we require a more permissive license like:

```suggestion
// SPDX-License-Identifier: MIT OR CC0-1.0
```